tools/vim/.ycm_extra_conf.py

```diff
@@ -0,0 +1,106 @@
+# A modified example from YCM main repository
```

Comment:
Please add copyright and license to all newly created files. Please see here for more information: https://github.com/STEllAR-GROUP/hpx/blob/master/.github/CONTRIBUTING.md